downstream/assemblies/eda/assembly-eda-credential-types.adoc

```diff
@@ -0,0 +1,15 @@
+[id="eda-credential-types"]
+
+= Credential types
+
+{EDAcontroller} comes with several built-in credental types that you can use for syncing projects, running rulebook activations, executing job templates through Automation Execution ({ControllerName}), uploading and downloading to container registries, and processing data through event streams. 
```

Comment:
@jself-sudoku This statement - uploading and downloading to container registries. Is not correct. EDA can only fetch images from container registries, it cannot upload anything.

Reply:
Thanks. I will update that.